filters/src/main/java/ua/net/tokar/json/rainbowrest/BasicHttpClient.java

```diff
@@ -0,0 +1,41 @@
+package ua.net.tokar.json.rainbowrest;
+
+import org.apache.http.Header;
+import org.apache.http.HttpEntity;
+import org.apache.http.client.methods.HttpGet;
+import org.apache.http.impl.client.HttpClients;
+import org.apache.http.message.BasicHeader;
+import org.apache.http.util.EntityUtils;
+
+import java.io.IOException;
+import java.net.URI;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class BasicHttpClient implements HttpClient {
+    private org.apache.http.client.HttpClient httpClient;
```

Comment:
seems like you don't need package name here

Reply:
We need it, because it's our HttpClient